### ISSUE TITLE
Kingdom - New Lands - CUSA10970

### ISSUE THREAD

**Issue by JohnGodgames** [red-prig/fpps4-game-compatibility#595]
# Game name
Kingdom - New Lands

## Code
CUSA10970

## Used emulator's version
[red-prig/fpPS4@](https://github.com/red-prig/fpPS4/commit/f243cfcfd58860e4198dbb498fc7a12620fb697d)

## Current status
Boots

## Error
```
EAssertionFailed: TODO tiling_idx:Thick_1dThick (vImageTiling.pas, line 498)
```

**Comment by GAMERRPCS3** [comment]
Retest

Current status - Ingame

Game code - CUSA10970

Game version - 01.01

Used emulator's version - 6e5a849

Used hacks - DEPTH_DISABLE_HACK - DISABLE_SRGB_HACK - SKIP_UNKNOW_TILING.


![1](https://user-images.githubusercontent.com/121258975/227390918-22408d04-4416-4f09-9200-0c8aba53f6d1.png)
![2](https://user-images.githubusercontent.com/121258975/227391008-9f82ece7-6568-43c3-9e5c-3571542ba433.png)

**Comment by GAMERRPCS3** [comment]
> For what you're using DEPTH_DISABLE_HACK and DISABLE_SRGB_HACK? They are improving something?

yes regarding, DEPTH_DISABLE_HACK. for graphics. and for DISABLE_SRGB_HACK born finally not nessecer.
and for SKIP_UNKNOW_TILING. This prevents the game from crashing on startup.

**Comment by GAMERRPCS3** [comment]
IMAGE_LOAD_HACK. after several tests the game has finally managed to arrive in the gameplay. with this option which gives better performance. but erases part of the character, on the horse.
